tools/styles/tokens/express/custom-large-vars.css

```diff
@@ -10,10 +10,26 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-/* This file contains overrides and additions to core tokens */
+/*!
```

Comment:
Thanks for pointing this out!

The `/*!` preventing existing tooling from removing that means that this is gonna get piled on a number of times across the project (a growing number as this decision makes its way across the CSS repo, the actually JS this specific file ends up in has three already just from this PR) and not wash out in the final production code. If Spectrum CSS feels it needs to ship this way, we'll likely build tooling to manually remove it so that our end consumers don't need to eat any size difference (regardless of how small the copyright is or what gzip might be able to do in any one file).

Reply:
Pretty standard for copyrights to use the important flag but also ultra easy to strip all comments using existing tooling: https://www.npmjs.com/package/postcss-discard-comments#user-content-removeall

My point however was not regarding this change - the Spectrum Web Components was already adding the copyright comment via tooling, I'm pointing out that the logic seems not to be catching that the copyright was _already_ there in the supplied file because it's perhaps not designed to handle the important flag.  The copyright has always been in these files, the only change was flagging it important.

Reply:
See above: https://github.com/adobe/spectrum-web-components/pull/3102/files#r1165738007.  This is now fixed.